### ISSUE TITLE
Feature Request: Add crps method to tfp.distributions

### ISSUE THREAD

**Issue by kaijennissen** [tensorflow/probability#1340]
Hi,

first of all I'd like to thank you for the great work.
I have a feature request for a new method which implements the 'continuous ranked probability score' for all distributions (for which it is possible, see f.e. [Evaluating Probabilistic Forecasts with scoringRules](https://arxiv.org/pdf/1709.04743.pdf)).
This method would enable the use of the `CRPS` as loss function in the same way as the negative log likelihood is used. (f.e. `crps_loss_fn = lambda y, p_y : p_y.crps(y) `)

Kind Regards
